js/templates/modals/listingDetail/listing.html

```diff
@@ -153,7 +153,7 @@ <h5><%= ob.polyT('listingDetail.paymentsAccepted') %></h5>
         <div class="verifiedMod">
           <div class="confirmBox clrBrAlert2 clrBAlert2Grad">
             <div class="flexVCent flexHCent gutterHTn rowTn">
-              <div class="badge" style="background-image: url(<%= ob.defaultBadge %>), url('../imgs/verifiedModeratorBadgeDefault.png');"></div>
+              <div class="badge" style="background-image: url(<%= ob.defaultBadge.tiny %>), url('../imgs/verifiedModeratorBadgeDefault.png');"></div>
```

Comment:
deafultBadge might be false, which would bomb this template.

Maybe it would be easiest to return the defaults from the defaultBadge function?

```
  defaultBadge(IDs) {
    const modelWithBadge = _.find(this.matched(IDs), mod => mod.get('type').badge);
    return {
      large: '../imgs/verifiedModeratorBadgeDefault.png',
      medium: '../imgs/verifiedModeratorBadgeDefault.png',
      small: '../imgs/verifiedModeratorBadgeDefault.png',
      tiny: '../imgs/verifiedModeratorBadgeDefault.png',
      ...(modelWithBadge && modelWithBadge.get('type').badge || {}),
    }
  }
```